src/components/NavigationSection.svelte

```diff
@@ -77,7 +76,7 @@
       }
     },
     computed: {
-      slug: label => slug(label)
+      hash: label => Math.abs([].reduce.call(label, (hash, char) => char.charCodeAt(0) + (hash << 5) + (hash << 16) - hash, 0))
```

Comment:
It's a simple an fast checksum algorithm initially created for sdbm (an OSS reimplementation of NDBM). I just wanted a simple way to generate a checksum without the need to embed an _md5_ or _sha_ module, and without rely on webcrypto which is clearly too complicated (promises and arraybuffers) for our need here: we just need a checksum based on the label after all :tongue:

This one is fast, any other can be as nice as this one, we can switch when / if we need later :smile: